web/modules/custom/hoeringsportal_content_blocks/templates/hoeringsportal-hearing-warning.html.twig

```diff
@@ -7,6 +7,7 @@
  * - config: The available texts
  */
 #}
+<div class="">
```

Comment:
Remove the empty `class` attribute.

Do we really need and empty `div` element if deadline is not passed? And do we need it to wrap the “danger” element if the deadline is passed?

Reply:
Thanks. Removed it.